### PR TITLE
[Merged by Bors] - chore: add docs + alias for `prod_bijective`

### PR DESCRIPTION
**Pull request by ericrbg**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/BigOperators/Basic.lean [hunk 1]
```diff
@@ -2008,7 +2008,7 @@ open Finset
 /-- `Fintype.prod_bijective` is a variant of `Finset.prod_bij` that accepts `Function.bijective`.
 
 See `Function.bijective.prod_comp` for a version without `h`. -/
-@[to_additive "`Fintype.sum_equiv` is a variant of `Finset.sum_bij` that accepts
+@[to_additive "`Fintype.sum_bijective` is a variant of `Finset.sum_bij` that accepts
 `Function.bijective`.
 
 See `Function.bijective.sum_comp` for a version without `h`. "]
```
Comment (eric-wieser) [inline Mathlib/Algebra/BigOperators/Basic.lean:2014]: Do you think we should rename this to `finset_sum_comp` to match?

**Review by eric-wieser — APPROVED** [review]
bors d+

Thanks!

File: Mathlib/Algebra/BigOperators/Basic.lean (hunk 1 above)
Comment (ericrbg) [reply Mathlib/Algebra/BigOperators/Basic.lean:2014]: I think they're similar and will get discovered through each others' names, which IMO is good enough. Maybe we should point out that `sum_comp` is for the case where `h` is `fun t => rfl`.

Comment (ericrbg) [reply Mathlib/Algebra/BigOperators/Basic.lean]: ah, that's what the "without `h`" means. I wonder if this one could get away with having `e` be implicit.

Comment (ericrbg) [reply Mathlib/Algebra/BigOperators/Basic.lean]: @eric-wieser just to triple-check you're happy with the current state before I `r+`

**Review by eric-wieser — APPROVED** [review]
bors merge

The renaming suggestion is really a follow-up suggestion anyway.

**Pull request closed without merging**